web/static/css/dfeed.css

```diff
@@ -610,7 +610,7 @@ a > .forcewrap {
 	margin: -15px 0;
 
 	font-family: "Roboto Slab", sans-serif;
-	font-size: 16px;
+	font-size: 15px;
```

Comment:
At 15px, the proportional text looks too small for me:
![](https://dump.cy.md/153050445843de613fcbd2c63233afb3/1617514211.213271296.png)

Reply:
15px looks much more consistent because it's the same size with non-formatted posts, you can see the same thing by changing it to 1em as well:
![image](https://user-images.githubusercontent.com/8372246/113500326-af299a80-9525-11eb-8f61-c50816dbfbb5.png)
Current 16px markdown posts are bigger than the non-formatted ones:
![image](https://user-images.githubusercontent.com/8372246/113500433-8e157980-9526-11eb-9dc6-28485e7a5d6a.png)